compose/.apps/prometheus/prometheus.yml

```
@@ -13,5 +13,6 @@ services:
     restart: ${PROMETHEUS_RESTART}
     volumes:
       - /etc/localtime:/etc/localtime:ro
-      - ${DOCKERCONFDIR}/prometheus:/config
+      - ${DOCKERCONFDIR}/prometheus:/etc/prometheus
+      - ${DOCKERSTORAGEDIR}/prometheus:/prometheus
```

```suggestion
      - ${DOCKERCONFDIR}/prometheus/etc:/etc/prometheus
      - ${DOCKERCONFDIR}/prometheus/data:/prometheus
```

Just using https://prometheus.io/docs/prometheus/latest/installation/#using-docker and https://github.com/selfhosters/unRAID-CA-templates/blob/cf265bcdcde5f6224098c183e2f69be734b9a898/templates/prometheus.xml#L16-L17 as a reference, I don't think the data from prometheus would be considered mass storage. Since the volumes were incorrect previously, I don't consider this a breaking change (it's a fixing change lol).

---

No strong opinion since I'm not that familiar with DockSTARTer, but it makes sense. Updated my PR!